st2reactor/st2reactor/rules/worker.py

```diff
@@ -48,6 +48,8 @@ def process(self, instance):
 
             if trigger_instance:
                 self.rules_engine.handle_trigger_instance(trigger_instance)
+            else:
+                LOG.info('No trigger found in db for instance. %s', instance)
```

Comment:
Being unable to create the trigger_instance implies that the trigger was not found? I guess if the check was in `create_trigger_instance` I would be more convinced :)